index.html

```diff
@@ -8,42 +9,37 @@
     <link rel="stylesheet" href="css/styles.css">
 </head>
 
-<body>
+<body class="d-flex">
 
     <header>
-        <nav class="navbar">
-            <div class="navbar logo">My Logo</div>
-            <button type="button" name="Menu"  class="navbar menu">
-                <img id="menu-button" src="img/icon/union.svg" alt="Menu Icon">
-            </button>
-            <!-- <ul class="navbar buttons">
-                <li id="Portfolio">Portfolio</li>
-                <li id="About">About</li>
-                <li id="Contact">Contact</li>
-            </ul> -->
-        </nav>
+        <div class="head-bar d-flex">
+            <a class="head-bar logo">My Logo</a>
+            <nav class="navbar">
+                <button type="button" name="Menu" class="navbar menu">
+                    <img id="menu-button" src="img/icon/union.svg" alt="Menu Icon">
+                </button>
+            </nav>
+        </div>
     </header>
 
     <div class="headline">
```

Comment:
- [ Optional] To be more semantic, this would be better as a section tag instead of a div

Reply:
Thank you!